agent/exec/container/container.go

```diff
@@ -83,9 +83,11 @@ func (c *containerConfig) image() string {
 }
 
 func (c *containerConfig) config() *enginecontainer.Config {
+	var hostname = strings.Replace(c.spec().Hostname, "*", fmt.Sprint(c.task.Slot), 1)
```

Comment:
Actually I need only fixed hostname on all containers. Would you accept such kind of UX?
I can easily revert my last commit in case fixed host name is acceptable.

Templated hostname will be useful for me if 'source' parameter of 'mount' option of docker service create is also templated.